lib/app.js

```diff
@@ -689,28 +699,12 @@ module.exports = {
                 .child('hosting/versions')
                 .child(settings.firebase)
                 .push();
-        var bar = null;
-        var total = 0;
         directoryRef.on('value', function(snapshot) {
           var status = snapshot.child('status').val();
-          if (status === 'deploying') {
```

Comment:
I see you removed the code for the "deploying" stage without replacing it with anything. Is this no longer needed or something?